src/main/java/models/Game.java

```diff
@@ -61,27 +55,26 @@ public void buildEnglishDeck() {
      */
 
     public void shuffle() {
-        long seed = System.nanoTime();
-        Collections.shuffle(deck, new Random(seed));
+        deck.shuffle();
     }
 
     public void dealFour() {
         for(int i = 0; i < 4; i++){
-            cols.get(i).add(deck.get(deck.size()-1));
-            deck.remove(deck.size()-1);
+            cols.get(i).add(deck.getTop());
+            deck.removeTop();
         }
     }
 
     //customDeal to setup game for testing purposes
     public void customDeal(int c1, int c2, int c3, int c4) {
-        cols.get(0).add(deck.get(c1));
-        deck.remove(c1);
-        cols.get(1).add(deck.get(c2));
-        deck.remove(c2);
-        cols.get(2).add(deck.get(c3));
-        deck.remove(c3);
-        cols.get(3).add(deck.get(c4));
-        deck.remove(c4);
```

Comment:
Lots of syntactical changes, going to pull down and check. It looks good though.